### PR TITLE
v2: Update README

### PR DESCRIPTION
**Pull request by benmoss**
cc @jmcshane

### CODE REVIEW

**Review by jmcshane — COMMENTED** [review]
Aren’t we using PodTemplate here instead of PodSpec? We will need to be able to set metadata such as labels and annotations

**Comment by jmcshane** [comment]
> Aren’t we using PodTemplate here instead of PodSpec? We will need to be able to set metadata such as labels and annotations

Sorry, this was meant to be inline with the Readme code, but GitHub made it a full review comment

File: README.md [hunk 1]
```diff
@@ -1,94 +1,125 @@
...
+kind: Pod
+metadata:
+  name: agent-abc
+spec:
+  initContainers:
+  - image: buildkite/agent:latest
+    args:
+    - start
+    - --acquire-job=abc
+    volumeMounts:
```
Comment (jmcshane) [inline README.md:82]: I know this is a future thing, but we might want to say something early on in the readme about “workspaces” and checking out the code into these volumes

Comment (benmoss) [reply README.md:87]: Added more description of the workspace in the "Explain it like I'm 5" section

**Pull request merged**